.stylelintrc

```diff
@@ -0,0 +1,135 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "color-named": "never",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-where-recommended",
+    "font-weight-notation": "numeric",
+    "function-url-quotes": "always",
+    "selector-max-specificity": "0,2,0",
+    "selector-no-id": true,
+    "selector-root-no-composition": true,
+    "string-quotes": "single",
+    "declaration-block-properties-order": [
```

Comment:
Mostly based on https://github.com/csscomb/csscomb.js/blob/master/config/yandex.json, but we can change if you think something looks very weird.